paddle/fluid/framework/ir/mkldnn_placement_pass.cc

```diff
@@ -34,4 +44,5 @@ std::unique_ptr<ir::Graph> MKLDNNPlacementPass::ApplyImpl(
 }  // namespace paddle
 
 REGISTER_PASS(mkldnn_placement_pass,
-              paddle::framework::ir::MKLDNNPlacementPass);
+              paddle::framework::ir::MKLDNNPlacementPass)
+     .RequirePassAttr("mkldnn_op_list");
```

Comment:
mkldnn_op_list -> mkldnn_enabled_op_types

the op list is just a list of op types, and better to be a `unordered_set`.

Reply:
Has changed, thanks.